docs/tutorials/jokes.md

```diff
@@ -7748,8 +7748,9 @@ Phew! And there we have it. If you made it through this whole thing then I'm rea
 [http-localhost-3000]: http://localhost:3000
 [bare-bones-hello-world-app]: /jokes-tutorial/img/bare-bones.png
 [remix-config-js]: ../file-conventions/remix-config
+<!-- page doesn't exist anymore -->
 [route-module]: ../route/route-module
```

Comment:
This page (https://remix.run/docs/en/dev/route/route-module) doesn't seem to exist anymore

Reply:
Same